musictheorpy/interval_utils.py

```diff
@@ -184,6 +184,16 @@ def descend_interval_from_name(self, qualified_interval_name):
         interval_bottom_note = _fetch_interval_bottom_note(self.rootnote, number_of_steps)
         return interval_bottom_note
 
+    def find_interval_from_root(self, root_note, top_note):
```

Comment:
Can we call this method "get_interval_name"? I think that name is more consistent with other places in the API.